benchmark/buffers/dataview-set.js

```diff
@@ -39,35 +39,35 @@ const mod = {
   setUint32: UINT32
 };
 
-function main({ millions, type }) {
+function main({ n, type }) {
   type = type || 'Uint8';
-  const len = millions * 1e6;
+
   const ab = new ArrayBuffer(8);
   const dv = new DataView(ab, 0, 8);
   const le = /LE$/.test(type);
   const fn = `set${type.replace(/[LB]E$/, '')}`;
 
   if (/int/i.test(fn))
-    benchInt(dv, fn, len, le);
+    benchInt(dv, fn, n, le);
   else
-    benchFloat(dv, fn, len, le);
+    benchFloat(dv, fn, n, le);
 }
 
-function benchInt(dv, fn, len, le) {
+function benchInt(dv, fn, n, le) {
```

Comment:
Renaming this variable is not necessary. I would rather change it to the way as it was before to reduce churn.

Reply:
In this `const len = millions * 1e6` was defined. There is no need of len as it's value is `n`. So that's why I replaced `len` with `n`. If I don't replace then I have to initialize the `len`

Reply:
So what I meant is that only the `main` function is impacted by that change. You can keep the `len` argument here and only change `len / 1e6` to `len`. That way there is less churn.

Reply:
that sounds good. Thanks alot